### ISSUE TITLE
adding time coordinate variables and attributes to NetCDF outputs

### ISSUE THREAD

**Issue by tobeycarman** [uaf-arctic-eco-modeling/dvm-dos-tem#317]
This has been on the list for a while, but integration with PEcAn has brought the issue to the surface.

Trying to maintain CF conventions. As it stands now, the first step is adding the appropriate data (and attributes) to the inputs, then we work on passing that thru to the outputs, filling out more data where necessary.

**Comment by tobeycarman** [comment]
@rarutter, feature or bug: when I specify both 'y' and 'm' for a single variable in the `output_spec.csv` file, I get only the monthly outputs? Is the idea if we have the monthly outputs, we could (as a post processing step) create the yearly outputs? Or should both files be generated?

**Comment by rarutter** [comment]
That is functioning as intended. Yeah, the thinking was that we could
reconstruct the larger granularity outputs from the finer if needed. If you
want both files, that can be changed, though.

**Comment by tobeycarman** [comment]
OK. glad it is working as expected. Now that I am working with it, I think it might be more intuitive if we changed it so that you get whichever outputs are specified in the output_spec file. And we have the output volume estimate working, so the user gets an up-front warning if they request too much stuff.

**Comment by rarutter** [comment]
Yeah, that would be more intuitive, and safer with the estimate. I don't think it rates implementing before the first test run, though (unless you can think of a reason we want the data separate).

**Comment by tobeycarman** [comment]
As of 11/25/2017, this issue is implemented/addressed for transient and scenario outputs.
